src/components/TodoContainer.js

```diff
@@ -0,0 +1,97 @@
+fetch('https://api.airtable.com/v1/myTable/myListItems?view=Grid%20view')
+  .then(response => response.json())
+  .then(data => {
+    // Handle the fetched data
+  })
+  .catch(error => {
+    // Handle any errors
+  });
+  fetch('https://api.airtable.com/v1/myTable/myListItems?view=Grid%20view&sort[0][field]=Title&sort[0][direction]=asc')
+  .then(response => response.json())
+  .then(data => {
+    // Handle the fetched data
+  })
+  .catch(error => {
+    // Handle any errors
+  });
+
+  .sort((objectA, objectB) => {
+    // Custom callback function for sorting
+  })
+  .sort((objectA, objectB) => {
+    const titleA = objectA.fields.Title.toUpperCase();
+    const titleB = objectB.fields.Title.toUpperCase();
+  
+    if (titleA < titleB) {
+      return 1; // Return 1 for descending order
+    }
+    if (titleA > titleB) {
+      return -1; // Return -1 for descending order
+    }
+    return 0;
+  })
+  const [sortDirection, setSortDirection] = useState('asc'); // Initial sort direction is ascending
+
+const handleToggleSort = () => {
+  setSortDirection(sortDirection === 'asc' ? 'desc' : 'asc');
+};
+fetch(`https://api.airtable.com/v1/myTable/myListItems?view=Grid%20view&sort[0][field]=Title&sort[0][direction]=${sortDirection}`)
```

Comment:
try to create constants for your base url that does not change- it makes your code more readable

Reply:
const BASE_URL = 'https://api.airtable.com/v1';
const TABLE_NAME = 'myTable';
const VIEW_NAME = 'Grid view';

const handleToggleSort = () => {
  setSortDirection(sortDirection === 'asc' ? 'desc' : 'asc');
};

fetch(`${BASE_URL}/${TABLE_NAME}/myListItems?view=${encodeURIComponent(VIEW_NAME)}&sort[0][field]=Title&sort[0][direction]=${sortDirection}`)
  .then(response => response.json())
  .then(data => {
    // Handle the fetched data
  })
  .catch(error => {
    // Handle any errors
  });

Would it be better